samples/drivers/led_xec/boards/mec15xxevb_assy6853.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+};
```

Comment:
What is the purpose for the parenthesis above ?

Reply:
not required, updated